### PR TITLE
🐛 fix: update broken icon links for provider Devicon

### PR DESCRIPTION
**Pull request by algovoid**
## What type of PR is this? 

- [x] Bug Fix

## What I did

updated broken links .

### CODE REVIEW

**Review by algovoid — COMMENTED** [review]
fixed the linting issue

**Comment by algovoid** [comment]
@maurodesouza

File: src/resources/tech_icons.ts [hunk 1]
```diff
@@ -3882,10 +3885,12 @@ const tech_icons = [
...
-        path: 'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain.svg',
+        path: 'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original.svg',
         variants: [
-          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain.svg',
-          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/laravel/laravel-plain-wordmark.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-original-wordmark.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-line.svg',
+          'https://cdn.jsdelivr.net/gh/devicons/devicon/icons/codepen/laravel-line-wordmark.svg',
         ],
```
Comment (maurodesouza) [inline src/resources/tech_icons.ts:3894]: The Laravel icons are still broken, and it seems to be due to the prefix codepen/. All other links follow the pattern brand/brand-icon.svg.

I believe it should be laravel/laravel-original.svg.

Could you please update this? Thanks!

**Review by maurodesouza — CHANGES_REQUESTED** [review]

**Comment by maurodesouza** [comment]
Hi @algovoid 👋

Sorry for the late answer!

I reviewed your PR and found an issue with the Laravel icon links!

**Review by algovoid — COMMENTED** [review]
should work now .

**Review by maurodesouza — APPROVED** [review]

**Pull request merged**